api/src/main/java/com/redhat/ipaas/api/v1/rest/CacheManagerProducer.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.ipaas.api.v1.rest;
+
+import org.infinispan.configuration.cache.Configuration;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.global.GlobalConfiguration;
+import org.infinispan.configuration.global.GlobalConfigurationBuilder;
+import org.infinispan.eviction.EvictionStrategy;
+import org.infinispan.manager.DefaultCacheManager;
+import org.infinispan.manager.EmbeddedCacheManager;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+
+@ApplicationScoped
+public class CacheManagerProducer {
+
+    @Produces
+    public EmbeddedCacheManager create() {
+        GlobalConfiguration g = new GlobalConfigurationBuilder()
+            .clusteredDefault()
+            .transport()
+            .clusterName("InfinispanCluster")
+            .build();
+
+        Configuration cfg = new ConfigurationBuilder()
+            .eviction()
+            .strategy(EvictionStrategy.LRU)
+            .maxEntries(150)
```

Comment:
Yes, we can use deltaspike's configuration value. But let's worry about it when we know what to do with the cache.